### PR TITLE
Fix/sd dashboard style

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

Update dashboards to use ui-kit tiles (based on [ui-kit-bs tiles](https://ovh-ux.github.io/ovh-ui-kit-bs/#tiles))

### Related PRs:

- [x] https://github.com/ovh-ux/ovh-ui-kit-bs/pull/52
- [x] Some changes will be needed when https://github.com/ovh-ux/ovh-manager-dedicated/pull/214 will be merged
- [x] https://github.com/ovh-ux/ovh-ui-kit-bs/pull/54
- [x] https://github.com/ovh-ux/ovh-ui-kit-bs/pull/58 should be merged
- [x] #416 should be merged
- [x] #337 should be merged
- [x] https://github.com/ovh-ux/ovh-ui-angular/commit/0927421ba0d72ba4849cae4d014612bbb9012a50 should be released

### Todo : 
- [x] remove `useUiKit` in service expiration component

### Reference:

SCDC-147

### CODE REVIEW

**Comment by JeremyDec** [comment]
@frenautvh translate adaptations are done on this branch

**Comment by FredericEspiau** [comment]
@cbourgois conflict :)

File: client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-    <h2 data-translate="cdn_tab_ssl"></h2>
-
     <div class="text-center"
          data-ng-show="loading">
         <oui-spinner></oui-spinner>
     </div>
-    <div class="row"
-         data-ng-show="ssl == null && !loading">
+
+    <div class="row d-md-flex mb-5" data-ng-show="ssl == null && !loading">
```
Comment (jleveugle) [inline client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html]: You get 2 times the same ng-show, i think it should be merged

File: client/app/components/expiration/service-expiration-date.component.controller.js [hunk 2]
```diff
@@ -14,6 +16,10 @@
             if (!hasValidServiceInfos || !hasValidServiceName) {
                 throw new Error("serviceExpirationDate: Missing parameter(s)");
             }
+
+            if (angular.isDefined(this.$attrs.useUiKit) && this.$attrs.useUiKit === "") {
```
[the remark this answers is not in the document]
Comment (cbourgois) [reply client/app/components/expiration/service-expiration-date.component.controller.js]: This was introduced to keep old style in dashboard during the migration to ui-kit-bs tiles style.

File: client/app/components/expiration/service-expiration-date.component.controller.js [hunk 3]
```diff
@@ -1,11 +1,13 @@
 (() => {
     "use strict";
     class ServiceExpirationDateComponentCtrl {
-        constructor ($scope, $rootScope, constants) {
+        constructor ($attrs, $scope, $rootScope, constants) {
```
[the remark this answers is not in the document]
Comment (cbourgois) [reply client/app/components/expiration/service-expiration-date.component.controller.js]: All theses parts should be removed before merge.

File: client/app/dedicated/server/dedicated-server.controller.js [hunk 4]
```diff
@@ -176,62 +169,6 @@ angular.module("App").controller("ServerCtrl", (NO_AUTORENEW_COUNTRIES, WEATHERM
         loadServer();
     });
 
-    $scope.editReverse = () => {
```
Comment (FredericEspiau) [inline client/app/dedicated/server/dedicated-server.controller.js]: It wasn't used ?

File: client/app/dedicated/server/display-name/display-name.controller.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+                }), _.get(data, "message")].join(". "), "server_dashboard_alert");
+                this.$state.go("^");
+            })
+            .finally(() => {
+                this.loading.update = false;
+            });
+    }
+
+    $onInit () {
+
```
Comment (FredericEspiau) [inline client/app/dedicated/server/display-name/display-name.controller.js]: Extra whitespace

File: client/app/dedicated/server/dedicated-server.controller.js (hunk 4 above)
Comment (cbourgois) [reply client/app/dedicated/server/dedicated-server.controller.js]: Nope, replaced by `app.dedicated.server.update-reverse-dns` state

File: client/app/license/spla/add/license-spla-add.html [hunk 6]
```diff
@@ -1,123 +1,118 @@
...
+        <div class="text-center"
+             data-ng-if="!$ctrl.options.availableServers">
+            <oui-spinner></oui-spinner>
+        </div>
 
-            <p class="text-danger"
-               data-ng-show="model.options.availableServers.length != null && !model.options.availableServers.length"
-               data-translate="license_spla_add_step1_server_unavailable">
+        <div data-ng-if="$ctrl.options.availableServers.length">
+            <p class="font-weight-bold"
```
Comment (FredericEspiau) [inline client/app/license/spla/add/license-spla-add.html]: No `strong`?

File: client/app/license/spla/add/license-spla-add.html [hunk 7]
```diff
@@ -1,123 +1,118 @@
...
-            </div>
+    <div data-wizard-step
+         data-wizard-step-on-load="$ctrl.loadTypes.bind($ctrl)"
+         data-wizard-step-valid="$ctrl.selected.type && $ctrl.selected.serial">
 
-            <p class="text-danger"
-               data-ng-show="model.options.availableTypes != null && model.options.availableTypes.length == 0"
-               data-translate="license_spla_add_step2_no_space_left">
-            </p>
+        <p class="font-weight-bold"
```
Comment (FredericEspiau) [inline client/app/license/spla/add/license-spla-add.html]: `strong` ?

File: client/app/license/spla/add/license-spla-add.html (hunk 6 above)
Comment (antleblanc) [reply client/app/license/spla/add/license-spla-add.html]: Used as an helper and applying margins as well.

File: client/app/license/spla/add/license-spla-add.html (hunk 7 above)
Comment (antleblanc) [reply client/app/license/spla/add/license-spla-add.html]: Used as an helper and applying margins as well.

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_ca#1141.0

* deploy_uxci
  * deploy_uxci ✔

**Comment by ovh-cds** [comment]
CDS Report build-yarn#1141.0

* Stage 1
  * deps ✔ 

* Stage 2
  * build ✔ 
  * test ✔

**Comment by ovh-cds** [comment]
CDS Report empty#1141.0

**Comment by antleblanc** [comment]
@cbourgois Existing a ticket related to this pull request? (If yes, could you please update the description with the ticket ID? thanks) And there are conflicts :)

**Pull request merged**